lib/vtls/schannel.c

```diff
@@ -2447,6 +2467,10 @@ static void schannel_session_free(void *ptr)
     if(cred->refcount == 0) {
       s_pSecFn->FreeCredentialsHandle(&cred->cred_handle);
       curlx_unicodefree(cred->sni_hostname);
+      if(cred->client_cert_store) {
+        CertCloseStore(cred->client_cert_store, 0);
+        cred->client_cert_store = NULL;
+      }
```

Comment:
```suggestion
#ifdef HAS_CLIENT_CERT_PATH
      if(cred->client_cert_store) {
        CertCloseStore(cred->client_cert_store, 0);
        cred->client_cert_store = NULL;
      }
#endif
```
according to 1592ea9 HAS_CLIENT_CERT_PATH checks for CryptStringToBinary because some builds like mingw which target versions of windows earlier than xp it might not be available, and I assume neither will CertCloseStore.  

also, client_cert_store declaration could be guarded the same way, however that would require moving the HAS_CLIENT_CERT_PATH logic into the EXPOSE_SCHANNEL_INTERNAL_STRUCTS section. i can do that or you can do it, you can see HAS_MANUAL_VERIFY_API for an example. basically we have to adjust for original mingw (if we are still supporting that even)

Reply:
Ah! That was my original intent, but I realized that the structure was declared before `HAS_CLIENT_CERT_PATH` would be available. I can restructure it a bit to follow what you did for `HAS_MANUAL_VERIFY_API`.

In the meantime, I'll also try to grab a build environment that will let me verify that it works properly.